lib/ui/mail_view.dart

```diff
@@ -0,0 +1,111 @@
+import 'dart:core';
+
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:summer2022/ui/bottom_app_bar.dart';
+
+class MailViewWidget extends StatelessWidget {
+  final FontWeight _commonFontWeight = FontWeight.w500;
+  final double _commonFontSize = 30;
+  final Color _buttonColor = Color.fromRGBO(51, 51, 102, 1.0);
+  final List<MailPiece> mailPieces = List.generate(
+      10,
+          (index) => new MailPiece(
+              timeStamp: DateTime.now(),
+              sender: "John Doe",
+              mailImage: Image.asset('assets/mail.test.02.png'),
+              mailDescription: "Lorem ipsum dolor sit amet, "
+              "consectetur adipiscing elit. "
+              "Sed consequat, quam non dictum volutpat, dolor odio sagittis dui, "
+              "quis dictum turpis nisl non ex. Donec suscipit euismod rutrum. ",
+              imageText: "Lorem ipsum dolor sit amet, "
+      )
+  );
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      bottomNavigationBar: const BottomBar(),
+      appBar: AppBar(
+        title: Text('Search Results',  style:
+        TextStyle(fontWeight: _commonFontWeight, fontSize: _commonFontSize),
+        ),
+        backgroundColor: _buttonColor,
```

Comment:
Did you mean to have this in thee TextStyle initiation above? it doesn't look like AppBar has a backgroundColor property.